projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/insurance/event-checkout.ts

```diff
@@ -76,11 +76,11 @@ export function checkOptionalProducts() {
 
 export function populatePersonalDetails() {
   cy.get('cx-dynamic-form').within(() => {
-    cy.get('[name="eventCountry"]').select('UK');
+    cy.get('[name="eventCountry"]').select('FR');
```

Comment:
Why changing this?

Reply:
Because UK is now GB, and since it may again change, FR is a safer bet...